devices/kintexu/familyInfo.xml

```diff
@@ -0,0 +1,5874 @@
+				<polarity_selector>
+					<name>REGCLK_OPTINV_CLKB_L</name>
+				</polarity_selector>
+			</corrections>
+		</site_type>
+		<site_type>
+			<name>FIFO36</name>
+			<is_fifo/>
+			<compatible_types>
+				<compatible_type>RAMBFIFO36</compatible_type>
+			</compatible_types>
+			<bels>
+				<bel>
+					<name>FIFO36E2</name>
+					<type>FIFO36E2_FIFO36E2</type>
+				</bel>
+			</bels>
+			<corrections>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKA_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKA_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKB_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKB_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENARDEN_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENARDEN_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENBWREN_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENBWREN_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_FIFORST</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_RSTREGA_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_RSTREGA_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>REGCLK_OPTINV_CLKA_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>REGCLK_OPTINV_CLKA_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>REGCLK_OPTINV_CLKB_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>REGCLK_OPTINV_CLKB_U</name>
+				</polarity_selector>
+			</corrections>
+		</site_type>
+		<site_type>
+			<name>RAMB36</name>
+			<is_bram/>
+			<compatible_types>
+				<compatible_type>RAMBFIFO36</compatible_type>
+			</compatible_types>
+			<bels>
+				<bel>
+					<name>RAMB36E2</name>
+					<type>RAMB36E2_RAMB36E2</type>
+				</bel>
+				<bel>
+					<name>HARD0GND</name>
+					<type>GND</type>
+				</bel>
+			</bels>
+			<corrections>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKA_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKA_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKB_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>CLK_OPTINV_CLKB_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENARDEN_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENARDEN_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENBWREN_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_ENBWREN_U</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_RSTRAMA_L</name>
+				</polarity_selector>
+				<polarity_selector>
+					<name>OPTINV_RSTRAMA_U</name>
+				</polarity_selector>
+	
```

Comment:
Not yet, but it is useful information that I would like to import into RapidSmith (I will make a TODO item).